components/Global/Navbar.jsx

```diff
@@ -1,90 +1,115 @@
   <a href={item.url}>
     <li
       className={`${
-        pathname === item.url && 'font-bold'
-      } ${classprops} cursor-pointer text-ml inline-block text-blue-800 no-underline hover:text-indigo-500 p-2 xl:px-4 items-center`}
+        pathname === item.url && "font-bold"
+      } ${classprops} cursor-pointer text-lg inline-block text-gray-500 no-underline hover:text-fuchsia-700 p-2 xl:px-4 items-center`}
     >
       {item.title}
     </li>
   </a>
-)
+);
 
 const Navbar = () => {
-  const router = useRouter()
-  const [toggleMenu, setToggleMenu] = useState(false)
+  const router = useRouter();
+  const [toggleMenu, setToggleMenu] = useState(false);
   const navItems = [
-    { title: 'HOME', url: prefix+'/' },
-    { title: 'LEARN', url: prefix+'#' },
-    { title: 'BLOG', url: prefix+'/blog' },
-    { title: 'ABOUT', url: prefix+'#' },
-    { title: 'PROJECTS', url: prefix+'/projects' },
-    { title: 'COMMUNITY PARTNERS', url: prefix+'/partners' },
-    { title: 'OUR MODERATORS', url: prefix+'/moderators' }
-  ]
+    { title: "Home", url: "/" },
+    { title: "About us", url: "#" },
+    { title: "Blog", url: "/blog" },
+    { title: "Learn", url: "#" },
+    { title: "Community Partners", url: "/partners" },
+    { title: "Projects", url: "/projects" },
+    { title: "Our team", url: "/moderators" },
+  ];
 
   return (
-    <nav className='sticky w-full z-30 top-0 text-white bg-white'>
-      <div className='w-full container mx-auto flex items-center justify-between mt-0 py-2'>
-        <div className='pl-4 flex items-center cursor-pointer'>
-          <Link
-            className='text-black no-underline hover:no-underline font-bold text-2xl lg:text-2xl'
-            href='/'
-            passHref
-          >
-            <img src={prefix+'/logo.png'} alt='WebXDAO' className='w-24 h-14 md:w-36 md:h-20'/>
-          </Link>
-        </div>
-        <ul className='lg:flex hidden list-none flex-row justify-end items-center ml-auto'>
-          {navItems.map((item, index) => (
-            <NavBarItem key={item.title + index} item={item} pathname={router.asPath} />
-          ))}
-        </ul>
-        <div className='flex relative p-2'>
-          {!toggleMenu && (
-            <HiMenuAlt4
-              fontSize={28}
-              className='lg:hidden cursor-pointer text-blue-600'
-              onClick={() => setToggleMenu(true)}
-            />
-          )}
-          {toggleMenu && (
-            <AiOutlineClose
-              fontSize={28}
-              className='lg:hidden cursor-pointer text-blue-600'
-              onClick={() => setToggleMenu(false)}
+    <nav className="sticky w-full z-30 top-0 bg-white flex text-gray justify-between items-center">
+      <div className="pl-4 my-2 flex items-center cursor-pointer">
+        <Link
+          className="no-underline hover:no-underline font-bold text-2xl lg:text-2xl"
+          href="/"
+          passHref
+        >
+          <a>
+            <Image
```

Comment:
Hi, we should'nt use nextjs/image due to SSR optimization. It'll break gh-pages deploy... Please use `<img src='' />` instead